### ISSUE TITLE
Style & settings adjustments

### ISSUE THREAD

**Issue by verovs** [telabotanica/moodle-theme_telaformation#26]
- [x] course/index.php : make all the courses category accordions open by default
- [x] course/index.php :  Lorsqu’on a la liste des mooc auxquels on peut s’inscrire : mettre le résumé avant de pouvoir aller vers la page d’accueil du cours en question. (the stuff showing under the "i" picto)
- [x] course/index.php : change of the logo before the course name (3 users/points/holding each others?) for the schoolar hat
- [x] course cards display on home only : hide the card footer (percent completion)
- [x] course cards display on home only  : Inscrire la période de diffusion du MOOC sous l’intitulé de la brique.
- [x] homepage carousel : when clicking on the sliders to slide, the focus is on blue > please delete the blue
- [x]  nav : when not connected, add a button with "connexion"  instead of "Non connecté (connexion) (see : https://mooc.tela-botanica.org/   )

- [x] settings: reactivate  the message thing  / and reactivate it also on the  user profile menu (drop-down in top right thing)
- [x] Finding and installing plugin to remove stuff in the left user menu, and removing : le calendrier , les fichiers personnels , la banque de contenu : local_boostnavigation   **In progress: installed but not fully configured, ok done**
- [ ] editorial stuff : reorganize categories names
- [x] homepage : button “se connecter/s’inscrire” bigger and in purple 
- [x] Changer le nom “cours disponibles” par “cours proposés”  (idk where)
- [x] remove the moodle logo (and link) on footer
- [x] reupload a cut version of the logo

**Comment by verovs** [comment]
@CrymSonir  I wrote all things here, do not hesitate to edit my broken english.
somme stuff are for me like  editorial stuff : reorganize categories names , is it ok if I let them in this list?

**Comment by verovs** [comment]
I do not find the setting for messaging where it should be in the online doc : admin/settings.php     section=optionalsubsystems

**Comment by CrymSonir** [comment]
Yes leave everything here I'll look inside, even if it means recreating tickets.
